security/pkg/server/ca/server.go

```diff
@@ -129,56 +129,6 @@ func recordCertsExpiry(keyCertBundle util.KeyCertBundle) {
 	certChainExpiryTimestamp.Record(certChainExpiry)
 }
 
-// HandleCSR handles an incoming certificate signing request (CSR). It does
-// proper validation (e.g. authentication) and upon validated, signs the CSR
-// and returns the resulting certificate. If not approved, reason for refusal
-// to sign is returned as part of the response object.
-// [TODO](myidpt): Deprecate this function.
```

Comment:
The PR LGTM, which is in line with the TODO comment here deprecating HandleCSR().